addons/im_livechat/static/src/legacy/widgets/feedback.js

```diff
@@ -130,11 +130,11 @@ var Feedback = Widget.extend({
                     uuid: this._livechat.getUUID(),
                     email: $email.val(),
                 }
-            }).then(function () {
-                self.$('.o_livechat_email').html($('<strong />', { text: _t('Conversation Sent') }));
-            }).guardedCatch(function () {
-                self.$('.o_livechat_email').hide();
-                self.$('.o_livechat_email_error').show();
+            }).then(() => {
+                this.$('.o_livechat_email').html($('<strong />', { text: _t('Conversation Sent') }));
```

Comment:
@seb-odoo ci/security fail because of this.

Reply:
I think it's not the first time I have to override this line, couldn't we start by changing it and fixing the template so that I don't have to do that at every step of the refactorign?

Reply:
We need much more refactoring so template is declarative _à la_ OWL. I don't see any decent workaround to avoid `ci/security` for the time being, whenever this LOC is changed (which will continue being affected in the future).